src/main/duke/commands/CBye.java

```diff
@@ -0,0 +1,18 @@
+package main.duke.commands;
+
+import main.duke.TaskList;
+import main.duke.Ui;
+import main.duke.enums.CommandType;
+
+public class CBye extends Command{
```

Comment:
Should the name of class be CommandBye? I think it is more meaningful. Similar with other command names.